lib/lightning_web/live/workflow_live/ai_assistant_component.ex

```diff
@@ -962,4 +971,8 @@ defmodule LightningWeb.WorkflowLive.AiAssistantComponent do
     user_messages = Enum.filter(session.messages, &(&1.role == :user))
     length(user_messages) > 1
   end
+
+  defp ai_feedback do
+    Application.get_env(:lightning, :ai_feedback)
```

Comment:
Hey @elias-ba, is this a new config? Is it a flag? Maybe we should add it to `Lightning.Config` instead

Reply:
The reason why I did it this way was to avoid introducing a new config variable in Lightning. We are trying to do the least possible to be able to inject the feedback component without defining how we inject it in Lightning.

Reply:
hey @midigofrank, thanks a lot for the clarifications. I have added the `:ai_feedback` config into the `config/config.exs`